client/finality-grandpa/src/environment.rs

```diff
@@ -378,14 +378,32 @@ impl<Block: BlockT> SharedVoterSetState<Block> {
 #[derive(Clone)]
 pub(crate) struct Metrics {
 	finality_grandpa_round: Gauge<U64>,
+	finality_grandpa_prevotes: Counter<U64>,
+	finality_grandpa_precommits: Counter<U64>,
 }
 
 impl Metrics {
-	pub(crate) fn register(registry: &prometheus_endpoint::Registry) -> Result<Self, PrometheusError> {
+	pub(crate) fn register(
+		registry: &prometheus_endpoint::Registry,
+	) -> Result<Self, PrometheusError> {
 		Ok(Self {
 			finality_grandpa_round: register(
 				Gauge::new("finality_grandpa_round", "Highest completed GRANDPA round.")?,
-				registry
+				registry,
+			)?,
+			finality_grandpa_prevotes: register(
+				Counter::new(
+					"finality_grandpa_prevotes",
+					"Total number of GRANDPA prevotes cast locally.",
+				)?,
+				registry,
+			)?,
+			finality_grandpa_precommits: register(
+				Counter::new(
+					"finality_grandpa_precommits",
```

Comment:
```suggestion
					"finality_grandpa_precommits_total",
```

Naming convention, see https://prometheus.io/docs/practices/naming/.